compositor_render/src/transformations/text_renderer.rs

```diff
@@ -193,48 +194,52 @@ impl TextRendererNode {
 
         buffer.set_wrap(font_system, text_params.wrap);
 
-        match text_resolution {
-            TextDimensions::Fixed { width, height } => {
-                buffer.set_size(font_system, width as f32, height as f32);
-
-                for line in &mut buffer.lines {
-                    line.set_align(Some(text_params.align));
-                }
-
-                buffer.shape_until_scroll(font_system);
-                (
-                    buffer,
-                    Resolution {
-                        width: width as usize,
-                        height: height as usize,
-                    },
-                )
-            }
+        let texture_size = match text_resolution {
+            TextDimensions::Fixed { width, height } => Resolution {
+                width: width as usize,
+                height: height as usize,
+            },
             TextDimensions::Fitted {
                 max_width,
                 max_height,
-            } => Self::layout_fitted(
-                buffer,
-                font_system,
-                Resolution {
-                    width: max_width as usize,
-                    height: max_height as usize,
-                },
-                &text_params,
-            ),
-            TextDimensions::FittedColumn { width, max_height } => Self::layout_fitted(
-                buffer,
-                font_system,
+            } => {
+                buffer.set_size(font_system, max_width as f32, max_height as f32);
+
```

Comment:
empty newlines can improve visibility to grup stuff, but if you add it between each line it has only negative effect